autosubmit/autosubmit.py

```diff
@@ -6067,4 +6107,4 @@ def retrieve_expids():
                     Log.info(f"Changing status of job {job.name} to {status}")
                     if status in Status.VALUE_TO_KEY.values():
                         job.status = Status.KEY_TO_VALUE[status]
-                job_list.save()
+                job_list.save()
```

Comment:
This last newline is useful to avoid `diff` issues - https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log